hexa/metadata/graphql/schema.graphql

```diff
@@ -0,0 +1,107 @@
+"""
+Interface for type implementing metadata
+"""
+interface HasMetadata {
```

Comment:
Having a variable name that represents a boolean looks like a bad idea. You can just call it `MetadataObject`